### ISSUE TITLE
Enhance the autoencoder model

### ISSUE THREAD

**Issue by josebadalmau** [Rulienn/Detection-d-anomalie#3]
Here are a few things you might want to try :
- Use cuda to accelerate the training process (and run tests faster).
- Decrease the learning rate, it seems it is set to 0.1, which might be a bit high.
- Try and use the Binary Cross-Entropy loss instead of the MSE.
- Add convolution layers to the architecture

**Comment by Rulienn** [comment]
-cuda is implemented
-We changed the learning rate to 1e-3
-We changed the loss function : loss_function = torch.nn.BCELoss()
-We are going to change some layers to convolution layers to uptade our model

**Comment by josebadalmau** [comment]
Great job, thanks !
